src/mitochondria/auth.ts

```diff
@@ -7,7 +7,7 @@ export interface ILoginResponse extends IAuth {
 }
 export interface ITokenResponse { access: string; }
 
-const BASE_URL = 'http://localhost:8000/' as const;
+const BASE_URL = 'https://murmuring-refuge-71505.herokuapp.com/' as const; // 'http://localhost:8000/' as const;
```

Comment:
Move this out to an environment variable.

`process.env.REACT_APP_BASE_URL`

Reply:
This should be made into an issue.

Reply:
I agree, this was more of a solution in wait for a decision on a generic solution for fetches and use of both dev-api and robohydra as host. This should be an issue.